MinVer/MinVer.targets

```diff
@@ -21,12 +21,16 @@
     </PropertyGroup>
   </Target>
 
-  <Target Name="MinVer" BeforeTargets="CoreCompile;GenerateNuspec" DependsOnTargets="MinVer_GetVersion" Condition="'$(DesignTimeBuild)' != 'true'">
+  <Target Name="MinVer" BeforeTargets="CoreCompile;GenerateNuspec" DependsOnTargets="MinVer_GetVersion" Condition="'$(UsingMicrosoftNETSdk)' == 'true' And '$(DesignTimeBuild)' != 'true'">
     <PropertyGroup>
       <MinVerVersion Condition=" '$(MinVerBuildMetadata)' != '' ">$(MinVerVersion)+$(MinVerBuildMetadata)</MinVerVersion>
       <Version>$(MinVerVersion)</Version>
       <PackageVersion>$(MinVerVersion)</PackageVersion>
     </PropertyGroup>
   </Target>
 
+  <Target Name="MinVer_Warnings" BeforeTargets="MinVer" Condition="'$(DesignTimeBuild)' != 'true'">
+    <Warning Condition="'$(UsingMicrosoftNETSdk)' != 'true'" Text="MinVer only works in Microsoft.NET.Sdk projects." />

```

Comment:
```suggestion
    <Warning Condition="'$(UsingMicrosoftNETSdk)' != 'true'" Text="MinVer only works in SDK-style projects." />
```

Reply:
I'm also wondering if this should be an error instead of a warning.

Reply:
I think it should be. Effectively, the error will be saying "this is wrong, fix it", which I think is reasonable, since there is no reason to have MinVer installed in a legacy project. I'll change it.